util/circular_buffer.c

```diff
@@ -146,12 +141,148 @@ int32_t cb_remove(struct circular_buffer *desc)
  *  - \ref SUCCESS : On success
  *  - \ref FAILURE : Otherwise
  */
-int32_t cb_size(struct circular_buffer *desc, uint32_t *nb_elements)
+int32_t cb_size(struct circular_buffer *desc, uint32_t *size)
```

Comment:
Update comment according to function signature.

Reply:
updated in #694